ghost/admin/app/styles/layouts/post-preview.css

```
@@ -60,6 +60,73 @@
+}
+
+.gh-post-preview-close .gh-post-preview-close-icon {
+    display: none;
+}
+
+.gh-post-preview-close .gh-post-preview-close-icon svg {
+    width: 14px;
+    height: 14px;
+    /* close.svg has no fill of its own; follow the button's text colour so it
+       stays visible on both the light and dark (theme-flipped) header */
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)